README.Rmd

```diff
@@ -73,18 +73,18 @@ install.packages("rtables")
 or you can install the latest development version directly from GitHub with:
 
 ```r
-remotes::install_github("insightsengineering/formatters")
-remotes::install_github("insightsengineering/rtables")
+# install.packages("pak")
+pak::pak("insightsengineering/rtables")
 ```
 
-Note you might need to set your `GITHUB_PAT` environment variable in order to be able to install from GitHub.
-
 Packaged releases (both those on CRAN and those between official CRAN releases) can be
 found in the [releases list](https://github.com/insightsengineering/rtables/releases).
 
+To understand how to use this package, please refer to the [Introduction to `rtables`](https://insightsengineering.github.io/rtables/main/articles/rtables.html) article, which provides multiple examples of code implementation.
```

Comment:
FYI: this link will be marked by url-checker until merged. After merge it would become green